nls/root/strings.js

```diff
@@ -11,7 +11,8 @@ define({
     COMMAND_UNNAMED: "Show Unnamed Functions",
     COMMAND_ARGS: "Show Arguments",
     COMMAND_INDENT: "Indent Entries",
-    COMMAND_AUTOHIDE: "Outline List Autohide",
+    COMMAND_AUTOHIDE: "Outline List (Enable Autohide)",
```

Comment:
I don't particularly like this renaming (and naming scheme). None of the other menu entries have parens.

I would maybe go with `Autohide Outline List` and `Set Outline List Autohide Delay`.

Reply:
Fixed.